docs/source/datamanagement/globus.md

```diff
@@ -65,22 +65,22 @@ After you have an account and set up a personal endpoint using Globus Connect pe
 
 ### Using the Northeastern endpoint
 
-To access the Northeastern endpoint on Globus, on the Globus web app, click **File Manager**, then in the **Collection** text box, type Northeastern. The endpoints owned by Northeastern University are displayed in the collection area. The general Northeastern endpoint is `Discovery Cluster`. This collection is a Managed Mapped Collection (GCS) hosted on dtn-03.rc.northeastern.edu. Using the File Manager interface, you can easily change directories, switch the direction of transferring to and from, and specify options such as transferring only new or changed files. Below is a procedure for transferring files from Discovery to your personal computer, but with the flexibility of the File Manager interface, you can adjust the endpoints, file view, direction of the transfer, and many other options.
```

Comment:
text: The general Northeastern endpoint is `Discovery Cluster` 
to be changed with: The general Northeastern endpoint is `Explorer Cluster`

Reply:
We need to confirm this one, since this endpoint it with Globus. May be we could come back to it once Explorer is more stable and the right endpoint is decided.